styleguide/components/page-review/page-review.stories.mdx

```diff
@@ -1,21 +1,33 @@
 import { Meta, Story, Preview, Source } from '@storybook/addon-docs/blocks';
-import { translate } from '../../story-helpers';
+import template from '../../examples/page_review/example.html';
 
 <Meta title="Components/Page review" />
 
 # Page review
 
-import template from './_example.html.haml';
-
 <Preview>
-  <Story
-    name="Example"
-    parameters={{ docs: { source: { code: template.raw } } }}
-  >
-    {(_, options) => translate(template, options)}
+  <Story name="Example" parameters={{ docs: { source: { code: template } } }}>
+    {() => template}
   </Story>
 </Preview>
 
+## Using with Rails
+
+If you are using the `citizens_advice_components` gem, you can call the component from within a template using:
+
+```rb
+= render(CitizensAdviceComponents::PageReview.new(page_review_date: Date.new(2020, 9, 21)))
+```
+
+Unlike the legacy haml partial the component accepts a date object rather than a formatted string. This allows us to define a default format. This can be customised by passing a `date_format` option.
```

Comment:
Would be nice to link to the `I18n` docs list of formats here (if there is one)?